federation/cluster/common.sh

```diff
@@ -87,7 +87,18 @@ function create-federation-api-objects {
       fi
     fi
 
-    export FEDERATION_SERVICE_CIDR=${FEDERATION_SERVICE_CIDR:-"10.10.0.0/24"}
+    if [[ "${KUBERNETES_PROVIDER:-}" == "gce" || "${KUBERNETES_PROVIDER:-}" == "gke" ]]; then
+      PD_PLUGIN_NAME='kubernetes.io/gce-pid'
```

Comment:
Surely this happens automatically on these providers?  Why would I ever want an aws-ebs on gce?  This is more a PVC question than a criticism of this PR.  I was not aware that PVC's needed to be told what provider to use. I would have thought that they would default to the native PV plugin on the provider.  @saad-ali @thockin ?